native/packages/react-native-bpk-component-progress/src/BkpProgress.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, View, Animated, Easing } from 'react-native';
+import {
+  animmationDurationBase,
+  spacingMd,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import STYLES from './styles';
+
+const styles = StyleSheet.create({
+  container: {
+    height: spacingMd,
+    overflow: 'hidden',
+  },
+  fill: {
+    height: spacingMd,
+  },
+});
+
+type Props = {
+  max: number,
+  min: number,
+  value: number,
+  type: 'Default' | 'Bar',
+  trackStyle: StyleSheet,
+  fillStyle: StyleSheet,
+};
+
+type State = {
+  progress: Animated.Value,
+  width: number,
+};
+
+class ProgressBar extends React.Component<Props, State> {
+  static defaultProps: Object;
```

Comment:
We can just add both `propTypes` and `defaultProps` right here instead of below the class. That's what we've stated doing for most components